src/picknik_ur_gazebo_config/objectives/loop_detect_april.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
The name "Loop Detect April" should be changed to "Loop Detect AprilTag" -- same with the corresponding file names / subtrees / etc.